src/processors/downgrade-unmodified-lines.js

```diff
@@ -0,0 +1,15 @@
+const fs = require( 'fs' );
+const downgradeUnmodifiedLines = require( '../lib/downgrade-unmodified-lines' );
+const differ = require( '../lib/differ' );
+const gitDiffCalculator = require( '../lib/git-diff-calculator' );
+
+const config = JSON.parse( fs.readFileSync( '.eslines.json', 'utf-8' ) );
+
+module.exports = function( report ) {
+	const remote = config.processors[ 'downgrade-unmodified-lines' ].remote || 'origin/master';
```

Comment:
This line with throw an exception when the processor is used for some branch, but the config property for `downgrade-unmodified-lines` is missing from the config file.

That's an error in the config file, of course, but it could be detected and reported instead of just crashing `eslines`.

It would be nice if each processor didn't have to load and parse the config file itself. What if it got the `config.processors[ processorName ]` as a second parameter, besides `report`.

This is a suggestion for further improvement in another PR -- it doesn't need to block this one from landing at all.